src/Workleap.OpenApi.MSBuild/ValidateOpenApiTask.cs

```diff
@@ -161,4 +177,16 @@ private async Task GeneratePublicNugetSource()
             File.WriteAllText(path, "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n<configuration>\n  <packageSources>\n    <clear />\n    <add key=\"nuget\" value=\"https://api.nuget.org/v3/index.json\" />\n  </packageSources>\n</configuration>");
         }
     }
+
+    // In the future, we will want to support both ADO and Github CI environments
+    private AdoCiReportRenderer InitializeCiReportRenderer()
+    {
```

Comment:
Right, we only support ADO environments so I've left this as AdoCiReportRenderer return value by default but once we migrate and support more products in Github, will update this accordingly